python/src/eiger_detector/control/eiger_detector.py

```diff
@@ -854,6 +855,36 @@ def lv_loop(self):
                 self.read_detector_live_image()
             time.sleep(0.1)
 
+    def hv_reset(self):
+        logging.info("Initiating HV Reset")
+
+        material = getattr(self, "sensor_material")
+        # Make sure sensor material is CdTe
+        try:
+            assert material.lower() == "cdte"
+        except AssertionError:
+            logging.info("Sensor material is not CdTe.")
+            return
```

Comment:
I meant something more like
```suggestion
       if material.lower() != "cdte"
           logging.error("Sensor material is not CdTe - Not running HV reset")
           return
```

and also replace the `raise Exception` below with something similar.